### DISCUSSION TITLE
how to load token to CURL POST request??

### DISCUSSION THREAD

**Discussion by NSH531** [CrowdStrike/falconpy#629] [Q&A]
I attached my exist  code

netanel 

```
@app.route("/load/<i>")
def loadertoken(i):
    falcon = falconpy.APIHarness(client_id="blah",client_secret="BLAH")
    x=R.api
    x=x.post("https://api.crowdstrike.com/oauth2/token",data={
        "client_id":"blah"
        ,
        "client_secret":"BLAH"
        })
    new_str = x.content.decode('utf-8') # Decode using the utf-8 encoding

    return json.loads(new_str)
@app.route('/s')
def SANDBOX():
    x=R.api.post("https://api.crowdstrike.com/samples/entities/samples/v2",loadertoken('123.123.123.123')["access_token"])
    new_str = x.content.decode('utf-8') # Decode using the utf-8 encoding

    return json.loads(new_str)
```

what's do I deed wrong

**Comment by NSH531** [comment]
@DimaWeissburd 
pls contribute on project

**Comment by jshcodes** [comment, marked as answer]
I'm a little confused on this one.   It appears we're posting directly to an API endpoint in the code above, but I'm not sure why we are doing so, so I'm having a hard time identifying how to assist.  (You do not need to reference endpoints in this fashion when making use of FalconPy.)

For the token value, you can always grab the value of your token using the Uber class with:

```python
token_value = falcon.token
```

If you've not authenticated before doing this, you will receive `False` as a response.  You can force authentication with:

```python
falcon.authenticate()
```
